crates/matrix-sdk-base/src/sliding_sync.rs

```diff
@@ -760,16 +760,18 @@ fn process_room_properties(
     }
 
     if let Some(recency_timestamp) = &room_data.timestamp {
-        room_info.update_recency_timestamp(*recency_timestamp);
-
-        // If it's not a new room, let's emit a `RECENCY_TIMESTAMP` update.
-        // For a new room, the room will appear as new, so we don't care about this
-        // update.
-        if !is_new_room {
-            room_info_notable_updates
-                .entry(room_id.to_owned())
-                .or_default()
-                .insert(RoomInfoNotableUpdateReasons::RECENCY_TIMESTAMP);
+        if room_info.recency_timestamp.as_ref() != Some(recency_timestamp) {
```

Comment:
I thought it wasn't possible for the server to send the same timestamp twice as any new event would change this value. Should we raise a bug onto the proxy?

Reply:
I think it might rather be that there's *another change elsewhere* in the room response subpart, but the timestamp might be the same, or something. I don't think we can blame the proxy for certainty here, maybe it's an expected behavior.